static/app/components/events/eventReplay/replayClipPreviewPlayer.tsx

```diff
@@ -0,0 +1,364 @@
+    );
+  }
+
+  if (replay.getDurationMs() <= 0) {
+    return (
+      <StaticReplayPreview
+        analyticsContext={analyticsContext}
+        isFetching={false}
+        replay={replay}
+        replayId={replayId}
+        fullReplayButtonProps={fullReplayButtonProps}
+        initialTimeOffsetMs={0}
+      />
+    );
+  }
+
+  return (
+    <PlayerContainer data-test-id="player-container" isLarge={isLarge}>
+      {replay?.hasProcessingErrors() ? (
+        <ReplayProcessingError processingErrors={replay.processingErrors()} />
+      ) : (
+        <ReplayPreviewPlayer
+          replayId={replayId}
+          fullReplayButtonProps={fullReplayButtonProps}
+          replayRecord={replayRecord}
+          issueCategory={issueCategory}
+          handleBackClick={handleBackClick}
+          handleForwardClick={handleForwardClick}
+          overlayText={overlayText}
+          showNextAndPrevious={showNextAndPrevious}
+          onClickNextReplay={onClickNextReplay}
+        />
+      )}
+    </PlayerContainer>
+  );
+}
+
+const PlayerPanel = styled(Panel)`
+  padding: ${space(3)} ${space(3)} ${space(1.5)};
+  margin: 0;
+  display: flex;
+  gap: ${space(1)};
+  flex-direction: column;
+  flex-grow: 1;
+  overflow: hidden;
+  height: 100%;
+`;
+
+const PlayerBreadcrumbContainer = styled(FluidHeight)`
+  position: relative;
+`;
+
+const PreviewPlayerContainer = styled(FluidHeight)<{isSidebarOpen: boolean}>`
+  gap: ${space(1)};
+  background: ${p => p.theme.background};
+
+  :fullscreen {
+    padding: ${space(1)};
+
+    ${PlayerBreadcrumbContainer} {
+      display: grid;
+      grid-template-columns: ${p => (p.isSidebarOpen ? '1fr 25%' : '1fr')};
+      height: 100%;
+      gap: ${space(1)};
+    }
+  }
+`;
+
+const PlayerContainer = styled(FluidHeight)<{isLarge?: boolean}>`
+  position: relative;
+  max-height: ${p =>
+    p.isLarge ? REPLAY_LOADING_HEIGHT_LARGE : REPLAY_LOADING_HEIGHT + 16}px;
+`;
+
+const PlayerContextContainer = styled(FluidHeight)`
+  display: flex;
+  flex-direction: column;
+  gap: ${space(1)};
+`;
+
+const StaticPanel = styled(FluidHeight)`
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+`;
+
+const StyledNegativeSpaceContainer = styled(NegativeSpaceContainer)<{isLarge?: boolean}>`
+  height: ${p => (p.isLarge ? REPLAY_LOADING_HEIGHT_LARGE : REPLAY_LOADING_HEIGHT)}px;
+  margin-bottom: ${space(2)};
+`;
+
+const ButtonGrid = styled('div')`
+  display: flex;
+  align-items: center;
+  gap: 0 ${space(2)};
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+const Container = styled('div')`
+  display: flex;
+  flex-direction: column;
+  flex: 1 1;
+  justify-content: center;
+`;
+
+const ContextContainer = styled('div')`
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: 1fr max-content max-content;
+  align-items: center;
+  gap: ${space(1)};
+`;
+
+const ReplayCellNoPadding = styled(ReplayCell)`
+  padding: 0 0 ${space(1)};
+`;
```

Comment:
No padding, or border padding?

I think we should just use ReplayCell directly and not have the padding.

```suggestion
```